tests/next_tests/integration_tests/cases.py

```diff
@@ -133,6 +133,13 @@ def field(
         sizes: dict[gtx.Dimension, int],
         dtype: np.typing.DTypeLike,
     ) -> FieldValue:
+        if hasattr(self.value, "__array__"):
+            return constructors.as_field(
+                common.domain(sizes),
+                np.full(tuple(sizes.values()), self.value),
+                dtype=dtype,
+                allocator=backend,
+            )
```

Comment:
which case triggers that actually?

Reply:
The cases where an array of boolean values is used as the value for ConstInitializer. Example: `test_gt4py_builtins.test_conditional_promotion`

Reply:
I don't understand that use case. What's the semantic? promote to 1 dimension more?

Reply:
It seems the existing use cases wer all erroneous and were not actually using arrays as point values.